src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureAbstract implements Figure {
+    private double firstLeg;
+    private double secondLeg;
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: " + getArea() + ", first leg: " + firstLeg + ", second leg: " + secondLeg + ", color: " + getColor());
+    }
+    }
+
+    @Override
+    public double getArea() {
+        return firstLeg * secondLeg;
```

Comment:
The formula for the area of a right triangle is incorrect. It should be `(firstLeg * secondLeg) / 2` to correctly calculate the area.